Cargo.toml

```diff
@@ -63,4 +63,4 @@ debug = false
 debug = false
 
 [patch.crates-io]
-webpki = { git = "https://github.com/linkerd/webpki", branch = "cert-dns-names-0.21", rev = "b2c3bb3" }
+webpki = { git = "https://github.com/linkerd/webpki", branch = "cert-dns-names-0.21" }
```

Comment:
did the pinned revision cause problems?

Reply:
```linkerd2-proxy/linkerd/proxy/http$ cargo +nightly fuzz build
error: failed to parse manifest at `/linkerd2-proxy-6/linkerd2-proxy/Cargo.toml`

Caused by:
  dependency (webpki) specification is ambiguous. Only one of `branch`, `tag` or `rev` is allowed.
Error: failed to build fuzz script: "cargo" "build" "--manifest-path" "linkerd2-proxy/linkerd/proxy/http/fuzz/Cargo.toml" "--target" "x86_64-unknown-linux-gnu" "--release" "--bins"```

Reply:
hmm, maybe we should prefer the pinned revision over the pinned branch, since the rev is more specific? but i don't have a strong opinion on this one.

Reply:
out of curiosity, is there a reason we don't use the latest of the branch?

Reply:
oh, huh, are we not? :)

Reply:
https://github.com/linkerd/webpki/commits/cert-dns-names-0.21 - just a single commit behind which changes the version.